R/recall_data.R

```diff
@@ -45,11 +68,28 @@ recall_by_make <- function(make, manufacturer = FALSE, limit = 25, partial = FAL
 #' \dontrun{
 #' recall_by_model('Civic')
 #' }
-recall_by_model <- function(model, limit = 25, partial = FALSE) {
+recall_by_model <- function(model,
+                           start_year = NULL, end_year = NULL,
+                           limit = 25,
+                           partial = FALSE) {
```

Comment:
is partial used yet?

Reply:
no, it's currently a placeholder related to filtering the output dataframe which @RyKoe  should be implementing (call to API returns all partial matches no matter what)